src/tenant/initialization.py

```diff
@@ -212,6 +214,17 @@ def create_initial_badges():
     ])
 
 
+def create_initial_menu_items():
+    MenuItem.objects.create(
+        label="Ranks List",
+        fa_icon="star-o",
+        url="/courses/ranks/",
```

Comment:
this shouldn't be hard coded, it should use `reverse('courses:ranks_list')` (or whatever that view is called) so it looks up the path in urls.py

If that's not possible (circular imports or some reason) Then need to add a test that this menu item's url is accessible (back up plan, reverse is better)

Reply:
url changed from hard-coded to reverse url.